test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -1113,6 +1113,36 @@ public async Task WaitForExternalEventWithTooLargeTimeout()
             }
         }
 
+        /// <summary>
+        /// End-to-end test which validates a CancellationToken-providing overload of WaitForExternalEvent.
+        /// </summary>
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task WaitForExternalEventWithCancellationToken()
+        {
+            var orchestratorFunctionNames = new[] { nameof(TestOrchestrations.ApprovalWithCancellationToken) };
+            var extendedSessions = false;
+            using (JobHost host = TestHelpers.GetJobHost(

```

Comment:
It looks like since you have written your code, `TestHelpers.GetJobHost()` returns an `ITestHost` instead of a `JobHost`. This is causing our CI pipeline to fail.

Reply:
Whoops!  Resolved. :)